android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/fields/DDMFieldCheckboxMultipleView.java

```diff
@@ -165,7 +165,9 @@ public void setParentView(View view) {
     public void setUpdateMode(boolean enabled) {
         if (this.field.isShowLabel()) {
             TextView label = findViewById(R.id.liferay_ddm_label);
-            label.setEnabled(enabled);
+	        if (label != null) {
+		        label.setEnabled(enabled);
+	        }
```

Comment:
Can we extract this code to a utility class? It is repeated in several places

Reply:
@victorg1991 Yes, we can extract this for a util class. I made that change right now. Thanks.